src/ColorField.php

```diff
@@ -13,19 +13,19 @@
 /**
  * @ORM\Entity
  */
-class ColorField extends Field implements Excerptable, FieldInterface
+class ColorField extends Field implements Excerptable, FieldInterface, Field\RawPersistable
```

Comment:
I'd suggest keeping these 'plain'

```suggestion
class ColorField extends Field implements Excerptable, FieldInterface, RawPersistable
```

And then adding a `use Bolt\Entity\Field\RawPersistable;` (from the top of my head)

Reply:
👍 Yes!

That doesn't quite fix the whole thing yet, not when creating a new record. I'll see what I can find :)